src/DynamoCoreWpf/Rendering/HelixRenderPackage.cs

```diff
@@ -57,10 +57,110 @@ public HelixRenderPackage()
+                                                                      org.X, org.Z, -org.Y, 1);
+
+
+            this.Transform = csAsMat.ToArray();
+        }
+
+        /// <summary>
+        /// Sets the transform that will be applied to all geometry in the renderPackage
+        /// by computing the matrix that transforms between from and to
+        /// </summary>
+        /// <param name="from"></param>
+        /// <param name="to"></param>
+        public void SetTransform(Autodesk.DesignScript.Geometry.CoordinateSystem from, Autodesk.DesignScript.Geometry.CoordinateSystem to)
+        {
+            var inverse = from.Inverse();
+            var final = inverse.PreMultiplyBy(to);
+
+            this.SetTransform(final);
+        }
+
+
+        /// <summary>
+        /// Sets the transform that will be applied to all geometry in the renderPackage
+        /// as this is a helix specific implementation it should be noted that the this matrix
+        /// should be as follows when converting from a ProtoGeometry/Dynamo CoordinateSystem
+        /// [ xaxis.X, xaxis.Z, -xaxis.Y, 0,
+        /// zaxis.X, zaxis.Z, -zaxis.Y, 0,
+        /// -yaxis.X, -yaxis.Z, yaxis.Y, 0,
+        /// org.X, org.Z, -org.Y, 1 ]
+        /// as Helix and Dynamo have their Y and Z axes reversed
+        /// </summary>
+        /// <param name="m11"></param>
+        /// <param name="m12"></param>
+        /// <param name="m13"></param>
+        /// <param name="m14"></param>
+        /// <param name="m21"></param>
+        /// <param name="m22"></param>
+        /// <param name="m23"></param>
+        /// <param name="m24"></param>
+        /// <param name="m31"></param>
+        /// <param name="m32"></param>
+        /// <param name="m33"></param>
+        /// <param name="m34"></param>
+        /// <param name="m41"></param>
+        /// <param name="m42"></param>
+        /// <param name="m43"></param>
+        /// <param name="m44"></param>
+        public void SetTransform(double m11, double m12, double m13, double m14,
+            double m21, double m22, double m23, double m24,
+            double m31, double m32, double m33, double m34,
+            double m41, double m42, double m43, double m44)
+        {
+            this.Transform = new System.Windows.Media.Media3D.Matrix3D(m11, m12, m13, m14,
+                m21, m22, m23, m24,
+                m31, m32, m33, m34,
+                m41, m42, m43, m44).ToArray();
+        }
+
+        /// <summary>
+        /// Sets the transform as a double array, this transform will be applied to all geometry in the renderPackage.
+        /// NOTE: this matrix is assumed to be in row vector form, and will be transformed into the neccesary form
+        /// for helix
+        /// </summary>
+        /// <param name="matrix"></param>
+        public void SetTransform(double[] matrix)
+        {
+            this.Transform = new double[]
```

Comment:
this is the change I am talking about, when clients use this interface they should know that if they provide their matrix in the row vector order specified the render package will take care of transforming it. what do you think?